serving/src/main/java/ai/djl/serving/http/AdapterManagementRequestHandler.java

```diff
@@ -89,7 +89,7 @@ protected void handleRequest(
                     handleListAdapters(ctx, decoder, modelName);
                     return;
                 } else if (HttpMethod.POST.equals(method)) {
-                    handleRegisterAdapter(ctx, decoder, modelName);
+                    handleRegisterAdapter(ctx, req, decoder, modelName);
```

Comment:
I think we should look into creating a requestParser, or leveraging the existing requestParser like we do in the InferenceRequestHandler. We can consolidate a lot of the code in that parser and leverage it for all the handle*Adapter methods

Reply:
Done